components/molecules/GalleryYoutubeVideo/index.tsx

```diff
@@ -0,0 +1,39 @@
+"use client";
+import { FC } from "react";
+import { VideoObject } from "schema-dts";
+import Skeleton from "react-loading-skeleton";
+import dynamic from "next/dynamic";
+import StructuredData from "@/components/atomic/StructuredData";
+
+const YouTubePlayerWithLoading = dynamic(
```

Comment:
the YouTube player doesn't play well with SSR, so it's loaded dynamically with a Skeleton fallback